lib/builders/dataTypes/emailAddressDataTypeBuilder.ts

```diff
@@ -0,0 +1,25 @@
+﻿import {DataTypeBuilder} from "./dataTypeBuilder";
```

Comment:
It seems that the InputType cannot be changed at all, so maybe it is not relevant to be able to input one.